### PR TITLE
feature/artist-profile-hero

### PR DESCRIPTION
**Pull request by coding-tomato**
## Trello ticket
[Ticket Link](https://trello.com/c/5Dst3si2)

## Description
Hero for different profiles.

## To reproduce
1. Run the project
2. Open the specific page (e.g. http://localhost:8080/[profile])
3. Check the component

## Screenshots
Mobile
image

Desktop
image

### CODE REVIEW

File: styles/base/_typography.scss [hunk 1]
```diff
@@ -8,79 +8,61 @@ body {
...
-        font-weight: 500;
-      }
+    b {
+      background-color: var(--pink);
+      font-weight: 500;
     }
   }
 }
 
 @media screen and (min-width: $tablet-viewport) {
```
Comment (SinuheM) [inline styles/base/_typography.scss]: $tablet-viewport have 768px then if you use mobile-first is necessary write +1 to have the correct viewport on tablet

File: styles/components/_hero.scss [hunk 2]
```diff
@@ -170,6 +211,18 @@
...
+        max-width: 533px;
+        padding: 42px 0 0 0;
+      }
+    }
+
+    &__button-wrapper {
+      display: flex;
+      justify-content: center;
+      margin: 48px 0 0 0;
+      width: 275px;
```
Comment (SinuheM) [inline styles/components/_hero.scss]: For this width the text at the button make a little linebreak

File: styles/base/_typography.scss [hunk 3]
```diff
@@ -8,79 +8,61 @@ body {
...
 
-      color: var(--tuscany);
-    }
-
-    &--child {
-      @include font-h3;
-
-      color: var(--tuscany);
+      b {
+        background-color: var(--pink);
```
Comment (SinuheM) [inline styles/base/_typography.scss:28]: The default weight on `b` = `bold` then here could have a `font-weight: 300` like his parent

image

File: styles/base/_typography.scss (hunk 1 above)
Comment (coding-tomato) [reply styles/base/_typography.scss]: Thank you Adrian, will get right to it soon.

**Review by SinuheM — APPROVED** [review]

**Review by dyepes62 — APPROVED** [review]

**Pull request merged**